iron-request.html

```diff
@@ -183,7 +183,7 @@
      *     async By default, all requests are sent asynchronously. To send synchronous requests,
      *         set to true.
      *     body The content for the request body for POST method.
-     *     headers HTTP request headers.
+     *     headers HTTP request headers. All keys must be lower case.
```

Comment:
Requiring this kind of constrains usage allowed by the RFC a bit: http://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html#sec4.2

>  Each header field consists of a name followed by a colon (":") and the field value. Field names are case-insensitive.